go.mod

```diff
@@ -9,19 +9,18 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
 	github.com/go-openapi/spec v0.19.4
-	github.com/golangci/golangci-lint v1.20.0 // indirect
 	github.com/google/go-cmp v0.4.0
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/open-cluster-management/api v0.0.0-20200623215229-19a96fed707a
 	github.com/open-cluster-management/multicloud-operators-deployable v0.0.0-20200625020633-0d458afa0fc2
 	github.com/open-cluster-management/multicloud-operators-placementrule v0.0.0-20200624201731-5e0510647fa9
```

Comment:
Please also refresh multicloud-operators-deployable and multicloud-operators-placementrule packages to get the latest version.

Reply:
fixed now.

Reply:
updated again to use latest
```
	github.com/open-cluster-management/api v0.0.0-20201007180356-41d07eee4294
	github.com/open-cluster-management/multicloud-operators-deployable v0.0.0-20201119200129-dcb15e7afa3f
	github.com/open-cluster-management/multicloud-operators-placementrule v1.0.1-2020-06-08-14-28-27.0.20201118195339-05a8c4c89c12
```